src/routes/api/sections/querySection.ts

```diff
@@ -0,0 +1,35 @@
+import { Request, Response } from 'express';
+import sectionModel from '../../../models/section.model';
+import timeblockModel from '../../../models/timeblock.model';
+
+const querySection = async (req: Request, res: Response): Promise<void> => {
+    // Retrieves the room section with the id given in the link
+    const sectionInformation = await sectionModel.findOne({_id: req.params.id}).catch((error) => {
+        res.status(404).json({ error });
+        return;
+    });
+    if(!sectionInformation) {
+        return;
```

Comment:
Let's return an error here - otherwise, the request will timeout and the client might be blocked for a few seconds.